AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/prefab_utils.py

```diff
@@ -346,13 +346,13 @@ def instantiate(self, parent_entity: EditorEntity=None, name: str=None, prefab_p
         container_entity_id = instantiate_prefab_result.GetValue()
         container_entity = EditorEntity(container_entity_id)
 
-        if name:
-            container_entity.set_name(name)
-
         wait_for_propagation()
 
         new_prefab_instance = PrefabInstance(self.file_path, EditorEntity(container_entity_id))
         assert not new_prefab_instance in self.instances, "This prefab instance is already existed before this instantiation."
```

Comment:
While you are here, can you fix the grammatical error here?
```suggestion
        assert not new_prefab_instance in self.instances, "This prefab instance already existed before this instantiation."
```

Reply:
Fixed.